test/spectests/spectests.cpp

```diff
@@ -289,6 +296,11 @@ class test_runner
                     pass();
                     continue;
                 }
+                catch (fizzy::validation_error const&)
+                {
+                    pass();
```

Comment:
The inverse of https://github.com/wasmx/fizzy/pull/241/commits/32cc345a2b296e6625a1a3dcab3ec0a1bac84d24#r398071288 applies here.

Reply:
This does not work.